server/services/crmDisplayService.test.ts

```diff
@@ -203,73 +125,76 @@ describe('CRM Display Service', () => {
+            firmPhone: '',
+            firmSupplierNo: '',
+            firmContactName: '',
+            firmSolicitorName: '',
+            firmSolicitorRef: '',
+          },
         },
         evidenceFiles: {
           files: [],
         },
-        FurtherInformation: {
-          files: [],
-        },
+        FurtherInformation: [],
       }
 
-      const result = crmDisplayService.getSections('crm5', 'general-information', customResponse)
+      const result = crmDisplayService.getSections('crm5', 'firm-details', customResponse)
 
       expect(result).toEqual([
         {
-          sectionId: 'general-information',
-          subsections: [{ fields: [], title: 'General Information' }],
-          title: 'General Information',
+          sectionId: 'firm-details',
+          subsections: [
+            {
+              title: 'Firm Details',
+              fields: [],
+            },
+            {
+              title: "Solicitor's Details",
+              fields: [
+                {
+                  apiField: 'Firm.firmAddress',
+                  label: 'Address/DX',
+                  type: 'multiline',
+                  value: '1 SOME PLACE',
+                },
+              ],
+            },
+            {
+              customDisplay: {
+                apiField: 'CaseDetails',
+                name: 'crm5CaseDetails',
+                value: '',
+              },
+              fields: undefined,
+            },
+          ],
+          title: 'Firm Details',
         },
       ])
     })
 
-    it('should handle crm section with no data in crm response', () => {
+    it('should skip sections with no data in crm response', () => {
       const result = crmDisplayService.getSections('crm5', 'court-of-appeal-funding', crm5Response)
 
-      expect(result).toEqual([
-        {
-          sectionId: 'court-of-appeal-funding',
-          subsections: [{ fields: [], title: 'Court of Appeal Funding' }],
-          title: 'Court of Appeal Funding',
-        },
-      ])
-    })
-
-    it('should return crm section for conditions met', () => {
-      const customResponse: Crm5Response = {
-        ...crm5Response,
-        formDetails: {
-          ...crm5Response.formDetails,
-          hasPreviousApplication: 'No', // ensure showWhen condition met
-        },
-      }
-
-      const result = crmDisplayService.getSections('crm5', 'capital-details', customResponse)
-
-      expect(result.length).toEqual(1)
-      expect(result[0].sectionId).toEqual('capital-details')
-      expect(result[0].title).toEqual('Capital Details')
+      expect(result).not.toContainEqual(expect.objectContaining({ sectionId: 'court-of-appeal-funding' }))
     })
 
     it('should return first section if conditions not met for given section', () => {
-      const result = crmDisplayService.getSections('crm5', 'capital-details', crm5Response)
+      const result = crmDisplayService.getSections('crm5', 'non-existent-section', crm5Response)
```

Comment:
Can wen change this back:-
```suggestion
      const result = crmDisplayService.getSections('crm5', 'capital-details', crm5Response)
```